tripy/tripy/frontend/module/sequential.py

```diff
@@ -12,30 +12,31 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Union, Tuple, Dict, Iterator, Any
-from dataclasses import dataclass
 import copy
+from dataclasses import dataclass
+from typing import Any, Callable, Dict, Iterator, Tuple, Union
 
 from tripy import export
 from tripy.common.exception import raise_error
 from tripy.frontend.module import Module
 
+ModuleLike = Union[Module, Callable[["tripy.Tensor"], "tripy.Tensor"]]
```

Comment:
This is just for convenience and is automatically expanded wherever relevant (e.g. the rendered docs)